samples/charts/tree-map/styling/App.razor

```diff
@@ -0,0 +1,41 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igComponent">
+
+        <Treemap Height="100%" Width="100%"
+                 DataSource="@Data"
+                 ParentIdMemberPath="Parent"
+                 IdMemberPath="Name"
+                 LabelMemberPath="Name"
+                 ValueMemberPath="Pop"
+                 LayoutType="TreemapLayoutType.Squarified"
+                 LayoutOrientation="TreemapOrientation.Vertical"
+                 FillScaleMode="TreemapFillScaleMode.Value"
+                 FillScaleMinimumValue="0"
+                 FillScaleMaximumValue="1500000000"
+                 FillBrushes="#4e62cf, #8a58d6"
+                 IsFillScaleLogarithmic="true"
+                 RootTitle="Countries"
+                 HeaderDisplayMode="@TreemapHeaderDisplayMode.Overlay"                  
+                 HeaderBackground="#E0F11F"                
+                 HeaderHoverBackground="#28E124"   
+                 Outline="black"                  
+                 StrokeThickness="1" 
+                 NodeStylingScript="onStyleTreeNode">@***refers to StylingTreemap.js***@
```

Comment:
per Graham's suggestion, try using node-style-mapping like this samples does:
https://codesandbox.io/s/eager-bash-zz47u?file=/index.html
this way you do not need to use JavaScript and make this app simpler

Reply:
markup doesn't work. I need to write up a bug. 

I updated the mappings to be added in in OnAfterRender